src/app/components/Footer/Footer.js

```diff
@@ -1,57 +1,40 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import logo from '../../../assets/logo.png';
+import appConfig from '../../../config/appConfig';
+import './Footer.scss';
 
 const Footer = props => {
   const { headerFooterVisibility } = props;
   const visible = headerFooterVisibility.value;
+  const renderLinks = arr =>
+    arr.map(link => (
+      <li key={link.id}>
+        <a href={link.href} rel="noopener noreferrer" target="_blank">
```

Comment:
Now we handle not origin links only. What about app links?
We should have a possibility to render router links.
eg: PLP link